app/src/main/AndroidManifest.xml

```diff
@@ -40,7 +40,6 @@
             android:exported="true"
             android:screenOrientation="portrait"
             android:theme="@style/AddoTheme.Login"
-            android:exported="true"
```

Comment:
Why are we removing this?

Reply:
@gosso22 It looks like a duplicate in a manifest file of the master branch. To verify the issue you can try to run the Addo app form the master branch